249 docs rfc make the grafana integration documentation into a single page doc (#3664)
timescale/docs: use-timescale/integrations/observability-alerting/grafana.md

---

> title: Getting started with Grafana and Timescale

Reviewer: ```suggestion
title: Integrate Grafana and Timescale Cloud
```

> excerpt: Use Grafana to visualize time-series data stored in Timescale

Reviewer: ```suggestion
excerpt: Use Grafana to visualize time-series data stored in a Timescale Cloud service
```

products: [cloud]
keywords: [Grafana, visualizations, analytics, monitoring]
---

import GrafanaConnect from "versionContent/_partials/_grafana-connect.mdx";

> # Get started with Grafana and Timescale

Reviewer: ```suggestion
# Integrate Grafana and Timescale Cloud
```

> Grafana is an analytics and monitoring tool that you can use to visualize time-series data. This page shows you how to set up Grafana with Timescale and create a dashboard and panel. It also provides an example of how to visualize geospatial data.

Reviewer: ```suggestion
You can use [Grafana](https://grafana.com/docs/) to monitor, visualize and perform analytics on data stored in your $SERVICE_LONG. 

This page shows you how to connect Grafana with a $SERVICE_LONG, create a dashboard and panel, then  visualize geospatial data.
```
